go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -533,6 +533,11 @@ var AlterVReplicationTable = []string{
 	"ALTER TABLE _vt.vreplication ADD COLUMN db_name VARBINARY(255) NOT NULL",
 }
 
+// AlterVReplicationTableSourceToBlob changes source column of vreplication table to BLOB
+var AlterVReplicationTableSourceToBlob = []string{
+	"ALTER TABLE _vt.vreplication MODIFY source BLOB NOT NULL",
```

Comment:
I think this should just be part of `AlterVReplicationTable`. The comment `// AlterVReplicationTable adds new columns to vreplication table` is misleading: `AlterVReplicationTable` is about all transformation post-CREATE.

Reply:
yea, i debated whether it belongs there. i figured we wanted to keep order of all migration steps...just in case

Reply:
If anything, putting all statements into the same array is the best guarantee for order.

Reply:
done

Reply:
i was wondering whether the `Alter` and `Create` should just be combined. i don't see any reason not to.

Reply:
Do you mean like the `CREATE` should be part of the array itself? I think that makes sense.